src/components/searchSection/catergrayList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { MDBIcon } from 'mdbreact';
+import './myStyle.scss';
+
+function CatergrayList({ items }) {
+  let itemsTags = [];
+  let numberOfItems = items.length;
```

Comment:
you don't have to make this as a variable it's obvious that `items.length` mean number of items